Installing-and-Configuring/OS/index.md

```diff
@@ -87,11 +85,11 @@ This model is vulnerable to brute force attacks but does not lock the data to th
 
 ### Security Dongle (aka smartcard aka security token)
 
-You may use a security dongle to manage the passphrase for the root LUKS partition.  This is also out of scope for heads even if using the same security dongle for verification such as in the [Pureboot](https://docs.puri.sm/PureBoot.html) model.  
+You may use a security dongle to manage the passphrase for the root LUKS partition.  This is also out of scope for heads even if using the same security dongle for verification.
 
-The first advantage to this approach is that the user only needs to type a PIN on boot rather than a passphrase.  Unlike the [heads TPM](#using-tpm-for-decryption-of-drives) model this model does not lock the data on disk to the hardware.  The hard drive may be moved to a different system and used with the security dongle and/or LUKS passphrase if it is still on the partition.  Also, the security dongle enforces limits which prevent brute force attacks.
+The first advantage to this approach is that the user only needs to type a PIN on boot rather than a passphrase.  Unlike the [heads TPM](#using-tpm-for-decryption-of-drives) model this model does not lock the data on disk to the hardware.  The hard drive may be moved to a different system and used with the security dongle and/or a LUKS passphrase.  Also, the security dongle enforces limits which prevent brute force attacks.
 
-In this model a large amount of text or binary data is used as the passphrase for decrypting the LUKS container.  This complex passphrase is encyrpted using the private key associated with the security dongle and placed in a 'public' location such as /boot.  After heads boots the Operating System the kernel and initrd use the security dongle to decrypt the passphrase which is used to decrypt the LUKS root partition.  
+In this model a large amount of text or binary data is used as the passphrase for decrypting the LUKS container.  This complex passphrase is encyrpted in a 'public' location such as /boot.  After heads boots the kernel it uses the security dongle to decrypt the passphrase which is then used to decrypt the LUKS root partition.
```

Comment:
As per precedent comment, the current Heads implementation is generating a [Disk Unlock Key of 128 bytes](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-seal-key#L59)

Consequently, this explanation needs to be rewritten.
The only real difference here between TPM based Disk Unlock Key and USB Security dongle's choice is that the implementation of proprietary USB Security dongle's smartcard vs TPM proprietary implementation, combined with the fact that the TPM stays in the computer, while the USB Security dongle is kept by the user. Another difference is that an eavesdropper seeing the USB Security dongle's User PIN typed at boot is predestined. A User having access to the USB Security dongle could consequently unlock Disk Encrypted content and encrypted emails, sign commits etc. While someone having access to the Disk Unlock Key passphrase can only decrypt the content on that computer having eavesdropped the passphrase at boot.

Also, the USB Security dongle's implementation is normally also linked to unlocking sessions if I remember well. So here, the USB Security dongle User PIN being known and having access to hit would give access to all private content (decrypt Disk encrypted content, unlock session, decrypt emails, impersonate user), where the TPM released Disk Unlock Key passphrase has limited impact to decrypting disk encrypted content. 

Another note is the the Disk Unlock Key passphrase is valid until core components of OS are updated, which will force Heads to prompt the user to set a new boot default, giving the user the possibility of changing the Disk Unlock Key passphrase, which requires only at that time the user to sign changed /boot digest.

Those are different use case for different threat models.
@jtmoree-github-com @kylerankin please clarify.

Reply:
also encyrpted -> encrypted
@jtmoree-github-com

Reply:
fixed spelling. rewrote sections.